### ISSUE TITLE
Sony memory card not recognized

### ISSUE THREAD

**Issue by shinamouri** [SKGleba/IMCUnlock#5]
After trying the plugin,  the sony memory card is no longer useable.  The vita appears to format it in settings,  but then requires a reboot,  which prompts for a format,  ending in failure and giving the following error: could not format.  (C1-2755-9). Safe mode format produces the same result.

**Comment by webvlad** [comment]
Have the same problem.
What i did:
Installed step by step.backups and others were made(hope they still safe on mc)
Then tried to install sd2vita and have several issues with use(via usb cant write to sd2vita) and decided to restore all back.
Made restore, disable ux0, everithing done.
But after reboot vita can't format mc and use it...

**Comment by SKGleba** [comment]
Fixed in v3.

**Comment by SKGleba** [comment]
Its a v2 bug, but lets try to find a fix. What fw are you guys on?

**Comment by webvlad** [comment]
On 3.68 :(

**Comment by shinamouri** [comment]
On firmware 3.65
    On Wednesday, August 8, 2018, 10:48:38 AM PDT, SKGleba <notifications@github.com> wrote:  
 
 
Its a v2 bug, but lets try to find a fix. What fw are you guys on?

**Comment by cufish** [comment]
Well I just triggered this using IMCUnlock v3... I guess my Vita is a brick now? :/ 

I created a 150Mb partition using IMCUnlockLite a couple days ago and wanted to get rid of it to downgrade (as suggested on modoru's readme) so I downloaded IMCUnlock and used the 0mb option (the readme of IMCUnlockLite said to do this to get rid of it permanently).

I was on 3.65 when this happened, I tried a bunch of stuff including factory resetting, formatting MC through settings and restore mode, but nothing worked. Out of frustration I updated to 3.68, but that didn't fix it either of course. Now I'm stuck on 3.68, can't use my official memory card (keeps asking to format and fails) so I can't install h-encore to even begin using my sd2vita again...
